src/components/card/card.tsx

```diff
@@ -89,9 +89,9 @@ type EuiCardProps = Omit<CommonProps, 'aria-label'> & {
   description: NonNullable<ReactNode>;
 
   /**
-   * Requires a <EuiIcon> node
+   * If provided, should be an `<EuiIcon>` node
```

Comment:
Another suggestion since the props table displays `null` as `enum` ☹️ 
```suggestion
   * Accepts an `<EuiIcon>` node or `null`
```